tests/data/shape.catalog.yaml

```diff
@@ -32,7 +61,8 @@ sources:
     driver: intake_geopandas.geopandas.ShapefileSource
     args:
       urlpath: simplecache::zip://gadm36_ALA_0*::https://biogeo.ucdavis.edu/data/gadm3.6/shp/gadm36_ALA_shp.zip
+      use_fsspec: True
       storage_options:
         simplecache:
-          same_names: true
+          same_names: True
```

Comment:
Any reason for capitalizing these? Both are okay, but lower-case is more true to YAML's JSON relationship

Reply:
no reason. changed back